sim/aviron/src/main.zig

```diff
@@ -176,6 +187,10 @@ const IO = struct {
     sp: u16,
     sreg: *aviron.Cpu.SREG,
 
+    // Exit status tracking
+    exit_requested: bool = false,
+    exit_code: u8 = 0,
```

Comment:
`exit_code: ?u8 = null,` stores both values in one element

Reply:
good call